packages/client/modules/userDashboard/components/GcalModal/GcalModal.tsx

```diff
@@ -164,14 +187,14 @@ const GcalModal = (props: Props) => {
   }
 
   return (
-    <Dialog isOpen={isOpen} onClose={closeModal}>
+    <StyledDialogContainer>
```

Comment:
I had to use the old modal instead of the radix one as the radix one wouldn't let me select any of the datetime fields. 

I tried updating the z-index in a few different places, and spent a fair amount of time looking into it, but the original modal works with the MUI datepicker so I think it's best to use that.

Reply:
Oh, that's unfortunate. Created an issue about this https://github.com/ParabolInc/parabol/issues/8711